src/compose/types/page-block/record-list.ts

```diff
@@ -22,6 +23,12 @@ interface Options {
   allowExport: boolean;
   perPage: number;
 
+  // Record-lines
+  inlineEditor: boolean;
```

Comment:
this is always implicit when editFields are there or any of draggable, positionField, parentField is set, correct?
anyhow, if it really needs to stay, `editable` might be better (so, same as `draggable`)